app/src/ui/lib/tooltip.tsx

```diff
@@ -7,6 +7,16 @@ import { assertNever } from '../../lib/fatal-error'
 import { rectEquals, rectContains } from './rect'
 
 export type TooltipDirection = 'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w' | 'nw'
+export enum TooltipDirectionDef {
+  NORTH = 'n',
+  NORTH_EAST = 'ne',
+  EAST = 'e',
+  SOUTH_EAST = 'se',
+  SOUTH = 's',
+  SOUTH_WEST = 'sw',
+  WEST = 'w',
+  NORTH_WEST = 'nw',
+}
```

Comment:
Why do we need two of these? Can't we just use the `enum`?

Or, at least, maybe build the `TooltipDirection` type using the enum values, no? 🤔

Reply:
.... Fear of regression. I just pushed a commit that updates 11 files to do this. :) I do prefer the enum everywhere.